test/integration_test.py

```diff
@@ -390,18 +406,21 @@ def _get_url(self,
                  allow_redirects: bool = True,
                  stream: bool = False
                  ) -> urllib3.HTTPResponse:
-        log.info('GET %s ...', url)
         retry = RetryAfter301(total=30, redirect=30 if allow_redirects else 0)
-        response = self._http.request('GET',
-                                      url,
-                                      retries=retry,
-                                      preload_content=not stream)
-        assert isinstance(response, urllib3.HTTPResponse)
-        log.info('... -> %i', response.status)
+        response = self._get_url_unchecked(url,
+                                           retries=retry,
+                                           preload_content=not stream)
         expected_statuses = (200,) if allow_redirects else (200, 301, 302)
         self._assertResponseStatus(response, expected_statuses)
         return response
 
+    def _get_url_unchecked(self, url: str, *args, **kwargs) -> urllib3.HTTPResponse:
```

Comment:
Given that the specification of the keyword argument is hard to find (it's the union of [this](https://urllib3.readthedocs.io/en/latest/reference/urllib3.response.html#urllib3.response.HTTPResponse) and [this](https://urllib3.readthedocs.io/en/latest/reference/urllib3.connectionpool.html#urllib3.HTTPConnectionPool.urlopen)) we should hardcode the keywrod args we actually use. It's only three AFAICT.